matchzoo/generators/point_generator.py

```diff
@@ -73,7 +75,7 @@ def _get_batch_of_transformed_samples(
         bsize = len(index_array)
         batch_x = {}
         batch_y = None
-        if 'label' in self.data:
+        if self.stage == 'train':
```

Comment:
you defined a `stage` variable in generator, how ever you didn't define any logic where `self.stage == 'test'`